src/payment/external-id-request-sender.ts

```diff
@@ -0,0 +1,25 @@
+import { RequestSender } from '@bigcommerce/request-sender';
+
+import { ContentType, INTERNAL_USE_ONLY } from '../common/http-request';
+
+export default class ExternalIdRequestSender {
```

Comment:
`SaveExternalId` seems to be a specific sub-endpoint of `/api/storefront/payment`

I think it would be more useful to consolidate this request sender to represent the main endpoint, and then the method represents the sub-endpoint, `saveExternalId(methodId, token)`

When used in a class, it will read clearer
```
storefrontPaymentRequestSender.saveExternalId(methodId, token)
```

I'd suggest calling it `StorefrontPaymentRequestSender` because we currently have a `PaymentRequestSender`, but that one interacts with BigPay.